SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -48,11 +49,12 @@ public class SonarQubeIssue
         /// </remarks>
         public string FilePath { get; }
         public string Hash { get; }
-        public int? Line { get; }
         public string Message { get; }
         public string ModuleKey { get; }
         public string RuleId { get; }
         public bool IsResolved { get; }
+        public SonarQubeIssueSeverity Severity { get; }
+        public IssueTextRange TextRange { get; }
         public IReadOnlyList<IssueFlow> Flows { get; }
```

Comment:
Technically there is also the Type field, but since it's needed only for Taint for now, we already know in SLVS that it's "vulnerability".